match/match_test.go

```diff
@@ -17,30 +21,35 @@ type mockComm struct {
 	userIDs []int64
 }
 
-func (md *mockDAO) ListMatch() (*[]dao.Match, error) {
-	matchList := make([]dao.Match, 0)
+func (md *mockDAO) ListMatch(input dao.ListMatchInput) (*[]dao.Match, error) {
+	mockMatchList := make([]dao.Match, 0)
 	for _, match := range md.matchList {
-		matchList = append(matchList, dao.Match{
-			ID:        match.ID,
-			UserOne:   match.UserOne,
-			UserTwo:   match.UserTwo,
-			MatchedOn: match.MatchedOn,
-		})
+		if match.AuthID == input.AuthID {
+			mockMatchList = append(mockMatchList, dao.Match{
```

Comment:
Confused why you're making a new object here? Why aren't you just appending the `match`?